pacman-nodejs-app/docs/pacman-nodejs-app-federated-aws-gke-portability.md

```diff
@@ -330,14 +321,13 @@ kubectl get rs mongo -o wide --watch
 We'll need to migrate the Pac-Man game to the us-west cluster.
 
 ```
-cat replicasets/pacman-replicaset-us-aws-gke-migration-2.yaml
-kubectl create -f replicasets/pacman-replicaset-us-aws-gke-migration-2.yaml
+kubectl annotate deploy/pacman federation.kubernetes.io/deployment-preferences='{"rebalance": true, "clusters": {"gce-us-west1": {"minReplicas": 2, "maxReplicas": 2, "weight": 0},"aws-us-east1": {"minReplicas": 0, "maxReplicas": 0, "weight": 0}}}'
```

Comment:
When specifying the annotations on the command line we are actually specifying the AWS cluster name, even if we are requesting 0 replicas. The file that we're removing, and replacing with this command, actually removed the AWS cluster name because unless it was gone from the annotations, the pacman replica was not being migrated. This clearly was a bug I was hitting with federation as it should have worked. Perhaps 1.6 has fixed it.